rtdata/languages/default

```diff
@@ -2815,7 +2815,8 @@ TP_LOCALLAB_BUTTON_DUPL;Duplicate
 TP_LOCALLAB_BUTTON_REN;Rename
 TP_LOCALLAB_BUTTON_VIS;Show/Hide
 TP_LOCALLAB_BWEVNONE;None
-TP_LOCALLAB_BWEVSIG;Sigmoid only
+//TP_LOCALLAB_BWEVSIG;Sigmoid only
+TP_LOCALLAB_BWEVSIG;Activated
```

Comment:
You could use GENERAL_ENABLED instead of creating a new key.

Reply:
Hello @Beep6581 
OK for delete instead of commented out.
For the new key,  I want to keep the possibility of a third choice (possible in locallabtools2)

Jacques